### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by anishchenko**
### html-css-popup

[Demo](https://anishchenko.github.io/Html-Css-Popup-Task/) |
[Code base](https://github.com/anishchenko/Html-Css-Popup-Task)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/anishchenko/html-css-popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+a {
+  display: block;
+  text-decoration: none;
+}
+
+.visualy_hidden:not(:focus):not(:active),
+input[type="checkbox"].visualy_hidden,
+input[type="radio"].visualy_hidden {
+  position: absolute;
+
```
Comment (HannaSyn) [inline submissions/anishchenko/html-css-popup/css/style.css:19]: Please, remove empty lines in blocks, you don't need them

File: submissions/anishchenko/html-css-popup/css/style.css [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+.popup__input:checked ~ .popup__list,
+.popup__input--more:checked ~ .popup__list--more {
+  display: flex;
+}
+
+/*  scroll */
+::-webkit-scrollbar { width: 0.3rem; height: 0.3rem;}
+::-webkit-scrollbar-button {  background-color: #e1eaea; }
+::-webkit-scrollbar-track-piece { background-color: #fff}
+::-webkit-scrollbar-thumb {background-color: #666; border-radius: 0.3rem;}
```
Comment (HannaSyn) [inline submissions/anishchenko/html-css-popup/css/style.css:216]: Please, keep one formatting style in whole file

File: submissions/anishchenko/html-css-popup/css/style.css [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+
+  display: none;
+} 
+
+.popup__input:checked ~ .popup__list,
+.popup__input--more:checked ~ .popup__list--more {
+  display: flex;
+}
+
+/*  scroll */
```
Comment (HannaSyn) [inline submissions/anishchenko/html-css-popup/css/style.css:212]: Avoid unnecessary comments

File: submissions/anishchenko/html-css-popup/css/style.css [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+::-webkit-scrollbar { width: 0.3rem; height: 0.3rem;}
+::-webkit-scrollbar-button {  background-color: #e1eaea; }
+::-webkit-scrollbar-track-piece { background-color: #fff}
+::-webkit-scrollbar-thumb {background-color: #666; border-radius: 0.3rem;}
+
+/* media */
+
+@media screen and (max-width: 36rem)  {
```
Comment (HannaSyn) [inline submissions/anishchenko/html-css-popup/css/style.css:220]: Why do you use relative units of measurement in media queries?

File: submissions/anishchenko/html-css-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+            <li class=" navigation__item navigation__item--popup">
+              <div class="popup">
+                <input class="popup__input" name="input-popup" id="popup__input" type="checkbox" >
+                <label class="popup__label" for="popup__input"></label>
+                <ul class="popup__list">
+                  <li class="popup__item">
+                    <a class="popup__link" href="#">
+                      <figure class="popup__figure">
+                        <img class="popup__img" src="img/icons/calendar.png" alt="calendar" width="40" height="30">
```
Comment (HannaSyn) [inline submissions/anishchenko/html-css-popup/index.html:32]: For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

**Review by HannaSyn — CHANGES_REQUESTED** [review]
Good work! 
Please, remove file submissions/anishchenko/html-css-popup/css/normalize.min.css, it is unnecessary for reiew.
See my comments below

**Comment by anishchenko** [comment]
done!
thanks for checking out the project

File: submissions/anishchenko/html-css-popup/css/style.css [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+  flex-direction: column;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+header {
+  margin-top: 0.8rem;
+}
+
+main {
```
Comment (HannaSyn) [inline submissions/anishchenko/html-css-popup/css/style.css:41]: You have classes `header` and `main`, why you don't use them in css?🙂

Comment (anishchenko) [reply submissions/anishchenko/html-css-popup/css/style.css]: i hope that i fix this problem)) thanks for review

**Pull request merged**